index.html

```diff
@@ -1,38 +1,206 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <link rel="stylesheet" href="styles.css"/>
   <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    </head>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="stylesheet.css"/>
+  <title>Portfolio</title>
+</head>
 <body>
-  <main class="Main-body">
-  
-    <header class="Portfolio-header">
+  <main class="container">
+      <header class="header_section">
       <nav>
-        <a href="#" target="_parent" class="portfolio-logo">My Portfolio</a>
-        <img src="./Images/Union-button.png" class="mobile-icon"/>
+        <a href="#" target="_parent" class="Portfolio_logo">My Portfolio</a>
+        <img src="./images/Union-button.png" class="mobile_menu_icon"/>
       </nav>
     </header>
-
     <section>
-      <div class="Intro">
-        <h1 class="Greeting">I’m Kamran Glad to see you!</h1>
-        <p class="Description">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>        
-           <a href="#" class="connecter">let's connect</a>
+      <div class="Greetings">
+        <h1 class="Intro">I’m Kamran Glad to see you!</h1>
+        <p class="Description">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,
+           don’t hestiate to contact me.</p>        
+           <a href="#" class="connect_button">let's connect</a>
```

Comment:
- [ ] Good job @Kamran123Ahmad, The one thing you should do here is to make the `Let's connect`  wrap in a heading tag `hx`. This will give it an emphasis when it comes to semantic.